docs/guides/cloud/data-storage-and-masking.mdx

```diff
@@ -0,0 +1,169 @@
+
+Add a control to your `cypress/support/index.js` file to set the default blackout options for all screenshots.
+
+```js
+before(() => {
+  Cypress.Screenshot.defaults({
+    // blackout all elements with 'mask-cy' attribute
+    blackout: ['mask-cy'],
+  })
+})
+```
+
+Add a control for a specific screenshot to blackout specific elements.
+
+```js
+it('admin page loads', () => {
+  cy.visit('/admin')
+  cy.screenshot({
+    blackout: ['.user-name'],
+  })
+})
+```
+
+#### Remove Cypress Command Log from screenshots
+
+Cypress offers an option to hide the Cypress Command Log from screenshots so that no information from that area will captured in screenshots. [See docs](/api/commands/screenshot#Arguments).
+
+Add a control to your `cypress/support/index.js` file to set the default capture options for all screenshots.
+
+```js
+before(() => {
+  Cypress.Screenshot.defaults({
+    capture: 'viewport', // or fullPage
+  })
+})
+```
+
+Add a control for a specific screenshot to only capture the application under test.
+
+```js
+it('sign in page', () => {
+  cy.visit('/sign-in')
+  cy.screenshot({
+    capture: 'viewport', // or fullPage
+  })
+})
+```
+
+### Video controls
+
+#### Disable video upload
+
+You can delete videos that are recorded when you are passing `video: true` by following [this guide](guides/guides/screenshots-and-videos#Control-which-videos-to-keep-and-upload-to-Cypress-Cloud). This prevents them from being uploaded to Cypress Cloud.
+
+### Cypress Command Log controls
+
+#### Turn off Cypress Command Log rendering
+
+You can disable showing the entirety of the Cypress Command Log as well as the AUT controls in your tests to prevent all test content from being shown and recorded in any screenshot or video by passing `--no-runner-ui` via the CLI while running Cypress tests. [See docs.](/guides/guides/command-line#Options). This means that the application under test will be the only thing that is rendered while running tests, so the only thing that is captured in screenshots and videos.
+
+Note that when `--no-runner-ui` is set, data from the Cypress Command Log will still be sent to Cypress Cloud to be generated and displayed in Test Replay if Test Replay is enabled. See [Disable Test Replay](#Disable-Test-Replay) for more information.
+
+#### Mask data in the Cypress Command Log
+
+There are multiple ways to mask data in the Cypress Command Log. Note that when these strategies are used, data within test definitions will still be sent to Cypress Cloud.
+
+##### `log: false` for Cypress commands
+
+You can pass `log: false` as an option to Cypress specific commands in your tests to prevent the data from being shown visibly in the Cypress Command Log and being shown in any captured screenshots or videos.
```

Comment:
```suggestion
You can pass `log: false` as an option to Cypress-specific commands in your tests to prevent the data from being shown visibly in the Cypress Command Log and being shown in any captured screenshots or videos.
```

Reply:
It may be useful to show a basic user/pw `.type()` scenario here in a code block to reiterate the `log: false` example IRL

Reply:
nm. I see we're showing this down below